### PR TITLE
[ci-visibility] Update javascript instrumentation to recommend `NODE_OPTIONS`

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
Change javascript to start recommending the usage of `NODE_OPTIONS`

### Motivation
Simplify the usage of the Datadog Library for CI Visibility. 

https://docs-staging.datadoghq.com/juan-fernandez/update-javascript-instructions/continuous_integration/setup_tests/javascript/?tab=jest#instrument-your-tests

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by albertvaka — APPROVED** [review]
LGTM

**Comment by juan-fernandez** [comment]
solving conflicts now

**Comment by juan-fernandez** [comment]
⚠️ Do not merge this yet. It's a significant change and I'd prefer to merge it on Monday

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 1]
```diff
@@ -87,93 +87,29 @@ yarn add --dev dd-trace
...
+{{< code-block lang="bash" >}}
+NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app yarn test
+{{< /code-block >}}
 
-```json
-"scripts": {
-  "test": "mocha --require dd-trace/ci/init"
-},
-```
+### Using `yarn>=2`
```
Comment (maycmlee) [inline content/en/continuous_integration/setup_tests/javascript.md:101]:
```suggestion
### Using Yarn >=2
```

**Review by maycmlee — CHANGES_REQUESTED** [review]
Left some feedback!

**Review by maycmlee — APPROVED** [review]

**Pull request merged**